lib/Lift.cpp

```diff
@@ -206,19 +212,39 @@ static void DefineNativeToLiftedWrapper(const remill::Arch *arch,
   // the spec could feasibly miss some dependencies, and so after optimization,
   // we'll be able to observe uses of `__anvill_reg_*` globals, and handle
   // them appropriately.
-  arch->ForEachRegister([=, &ir](const remill::Register *reg_) {
-    if (auto reg = reg_->EnclosingRegister(); reg_ == reg) {
-      std::stringstream ss;
-      ss << "# read register " << reg->name;
+  if (FLAGS_feature_inline_asm_for_unspec_registers) {
```

Comment:
Can you move this `if` statement to be inside the `arch->ForEachRegister`, following the definition of `ss`.

Reply:
Pushed updated version that moves the feature flag check inside the loop